webrtc-stats.html

```diff
@@ -356,6 +356,39 @@ <h3>
         </p>
       </section>
     </section>
+    <section>
+      <h2>
+        Privacy considerations
```

Comment:
This needs to have a reason to go here rather than in "security and privacy considerations" at the end of the document.

I suggest using the title "Procedures for mitigating privacy concerns" and adding a reference to this section in the "Security and privacy considerations" section.

Reply:
Fixed in follow-up: https://github.com/w3c/webrtc-stats/pull/697